lang/docs/templates/module.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>{{title}}</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<div>
+        <h1>{{title}}</h1>
+        <pre>
```

Comment:
[`pre` is whitespace-sensitive](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre), so better change to `<pre><code>{{code}}</code></pre>`. Otherwise it may render additional leading/trailing whitespace.

Reply:
This can be seen, for example in the example I gave above.